### DISCUSSION TITLE
Wegpunkte im Kurs Editor

### DISCUSSION THREAD

**Discussion by PhilippS1** [Courseplay/Courseplay_FS22#2807] [Q&A] [answered]
Hallo zusammen,

wie kann ich einem Wegpunkt die Option Werkzeug senken oder heben hinterlegen? Ich habe einen Automatisch generierten Kurs welchen ich jetzt noch händisch abändern will. Nun habe ich jedoch das Problem das der Wegpunkt wohl noch im Hintergrund weiß wie die Position des Werkzeuges ursprünglich war. 
Was zur Folge hat das der Helfer zwar am Wegpunkt das Werkzeug senkt jedoch am nächsten wieder anhebt und mit angehobenen Werkzeug weiterfährt bis zum nächsten grünen Dreieck.

Gruß

**Comment by Tensuko** [comment, marked as answer]
Das gibt es nicht. Das wird alles über turn start und turn end geregelt.

Alle manuellen Änderungen am Kurs geschehen auf eigenes Risiko :)

**Reply by PhilippS1** [reply]
Also wenn ich dich richtig verstehe auf grün heben und auf rot senken?

**Reply by Tensuko** [reply]
Es gibt kein heben und senken.
Es gibt nur Anfang eines Wendemanövers und Ende.
Alles was dazwischen passiert hängt vom tool, der Richtung der Wegpunkte usw ab.

**Reply by PhilippS1** [reply]
![Wegpunkt](https://github.com/Courseplay/Courseplay_FS22/assets/55155040/03f691f6-5f64-4470-a318-2b76d483d85d)

Dem zufolge müsste er dann jetzt weiter arbeiten oder? Alle Wegpunkte nach dem dem roten Wegpunkt habe ich manuell gesetzt.

**Reply by Tensuko** [reply]
Ohne deinen ganzen kurs zu kennen kann ich dir nichts sagen.
Außerdem solltest du nur die WP modifizieren wenn du weist was du tust. Gleiche anzahl turn start wie turn end.

**Reply by Tensuko** [reply]
Bevor das ein hin und her wird, was willst du überhaupt erreichen?

**Reply by PhilippS1** [reply]
Der Helfer fährt ordnungsgemäß vom Start Wendepunkt zum Ende Wendepunkt. Senkt den Pflug fährt über den ersten Wegpunkt nach dem Wendepunkt, hebt dann den Pflug wieder an und fährt die Wegpunkte mit angehobenen Pflug ab bis er wieder beim nächsten Start Wendepunkt ankommt.

Ich will aber das der Helfer nach dem Ende Wendepunkt regulär weiterarbeitet bis zum Start Wendepunkt

**Reply by Tensuko** [reply]
Dann generiere einen Kurs und guck dir an wie sowas aus zu sehen hat...

**Reply by Tensuko** [reply]
Alle manuellen Änderungen am Kurs geschehen auf eigenes Risiko

**Reply by PhilippS1** [reply]
Der Kurs ist generiert. Daher weiß ich wie er auszusehen hat. Natürlich geschehen alle manuellen Änderungen auf eigenes Risiko. 
Wenn du sagst es gibt gibt nur drei Arten von Punkten, den Start Wendepunkt, den Ende Wendepunkt und den Wegpunkt dann erkläre mir mal bitte. Wieso der Helfer nach dem bearbeiten der Bahnen das Gerät anhebt den Wegpunkten folgt und bis zum Anfang des Vorgewendes fährt ohne über eine Start oder Ende Wendepunkt zu fahren. Demzufolge müssen die Wegpunkte eine zusätzliche Information beim generieren erhalten bzw. es gibt zwei Unterschiedliche Wegpunkte.

**Reply by Tensuko** [reply]
Gibt es nicht!
Wenn er das tool nicht wieder absenkt hat es andere gründe.

**Reply by Tensuko** [reply]
Es gäbe noch transition aber die kann ein user nicht einbauen. Und da ich nicht weis was du da selber wann wo eingefügt hast kann ich dir auch nicht helfen.

Generiere einen komplett neuen Kurs und probiere es aus, wenn es dann Probleme gibt, muss was mit der tool kombi sein.

**Reply by PhilippS1** [reply]
Also gut mir reicht es. Du hast mir meine Frage über Umwege beantwortet. Es ging mir nicht darum um etwas zusätzlich einzubauen sonder darum warum er so Tickt wie er Tickt. Die Tool Kombi funktioniert, daher hatte ich den begründeten Verdacht das sich Couseplay beim generieren der Wegpunkte mehrere Informationen im Hintergrund merkt. 
Danke für deine Mühe.

**Reply by PhilippS1** [reply]
Danke für deinen Tipp, habe jetzt den Kurs nochmals umgebaut. Der einzige Unterschied ist das ich für den Beginn meines manuell Angelegten Kursabschnittes einen Wegpunkt verwendet habe den Courseplay beim generieren das Gerät abgesenkt hatte.

**Reply by Geri-G93** [reply]
es gibt noch einen 4ten Wegpunkt typ, ConnectingTrack
da hebt er an, hast du den in eine Bahn gezogen macht er genau die sachen
